django_mesh/models.py

```diff
@@ -191,4 +194,48 @@ class MockYoutube(models.Model):
     you = models.CharField(max_length=140, unique=True)
     yoh = models.CharField(max_length=140, unique=True)
     yot = models.TextField(default='')
-    yoj = models.TextField(default='')
\ No newline at end of file
+    yoj = models.TextField(default='')
+
+class Media(_Abstract):
+
+    MEDIA_TYPE = Choices(
+        (0, 'NONE',       'none',),
+        (1, 'IMAGE',      'image',),
+        (2, 'VIDEO',      'video',),
+    )
+
+    channel = models.ForeignKey(Channel)
+    objects = MediaQuerySet.as_manager()
+
+
+    upload_file = models.FileField(upload_to='uploads/%Y/%m/%d')
+
+    media_type = models.IntegerField(max_length=1, default=MEDIA_TYPE.NONE, choices=MEDIA_TYPE)
+
+
+
+    oembed_html = models.TextField(default='', blank=True)
+
+    @property    
+    def get_file_url(self):
```

Comment:
Since this is a property, we can just call it "file_url", since it gets called like an attribute and not a function. Normally "get_" is used for functions.